Code/ThirdParty/Imgui/imconfig.h

```diff
@@ -18,7 +18,7 @@
 // Start ezEngine edit
 // Configure the DLL Import/Export Define
 #ifdef BUILDSYSTEM_COMPILE_ENGINE_AS_DLL
-#ifdef WIN32
+#ifdef _WIN32
```

Comment:
These are third party libs, are you sure this is incorrect?

Reply:
Maybe we should define the proper stuff in cmake, rather than change this code.

Reply:
Yes this is correct, the issue only existiert due to a ezEngine modification of this lib (same as the others). So I fix the ezEngine edit.

Reply:
Some further details. The `WIN32` define is not standardized. It is only defined if you build from within visual studio. The `_WIN32` define on the other hand is build into the compiler (both clang and msvc). See https://learn.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170
So it's cleaner to use `_WIN32` instead of defining `WIN32` ourselfs.